CMakeLists.txt

```diff
@@ -189,5 +191,6 @@ if (WIN32)
     target_link_libraries(snowflakeclient)
 endif ()
 
+add_subdirectory(cpp)
```

Comment:
I think this was already in main. Please rebase.

Reply:
I think so as well. I wasn't able to rebase my branch with master, I had to merge it in. When this gets merged into the master branch, it can be done as part of a squash commit.